config.toml.example

```diff
@@ -1,34 +1,22 @@
 # A sample configuration for fedora-messaging. This file is in the TOML format.
-# For complete details on all configuration options, see the documentation.
-
 amqp_url = "amqp://"
-
-publish_exchange = "amq.topic"
-
 callback = "fedora_messaging.example:printer"
-
-# The topic_prefix configuration value will add a prefix to the topics of every sent message.
-# This is used for migrating from fedmsg, and should not be used afterwards.
+passive_declares = false
+publish_exchange = "amq.topic"
 topic_prefix = ""
 
-# Note the double brackets below.
-# To add another binding, add another [[bindings]] section.
-[[bindings]]
-queue = "my_queue"
-exchange = "amq.topic"
-routing_keys = ["#"]
-
 [tls]
-ca_cert = "/etc/pki/tls/certs/ca-bundle.crt"
-keyfile = "/my/client/key.pem"
-certfile = "/my/client/cert.pem"
+ca_cert = "/etc/fedora-messaging/fedora.cacert.pem"
```

Comment:
In the other example config files, this file is called `cacert.pem`. Is it intended to have a different name here?